.github/container/nsys_jax/nsys_jax/scripts/install_protoc.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import google.protobuf
+import io
+import os
+import platform
+import requests
+import zipfile
+
+
+def main():
+    # TODO: add a default to (with confirmation) install in the same prefix as this script is installed to
+    parser = argparse.ArgumentParser(
+        "Install a version of the protoc compiler that is compatible with the google.protobuf runtime"
+    )
+    parser.add_argument(
+        "prefix", help="Output prefix under which to install protoc", type=str
+    )
+    args = parser.parse_args()
+
+    s = requests.Session()
+    s.mount("https://", requests.adapters.HTTPAdapter(max_retries=5))
+
+    # protobuf versioning is complicated, see protocolbuffers/protobuf#11123 for more
```

Comment:
Didn't this logic already use to live somewhere else? I don't recall where though.

Reply:
It moved from here: https://github.com/NVIDIA/JAX-Toolbox/pull/1165/files#diff-634c5bb0e37da5e8d8aaebb04448e80c62f14a16b902a3e9931a7054038bcdb0L21-L32
This file is mainly a rename of `install-protoc`.